lib/winston/transports/console.js

```diff
@@ -30,6 +31,20 @@ module.exports = class Console extends TransportStream {
     this.name = options.name || 'console';
     this.stderrLevels = this._stringArrayToSet(options.stderrLevels);
     this.consoleWarnLevels = this._stringArrayToSet(options.consoleWarnLevels);
+    this.consoleLevels = options.consoleLevels || {
```

Comment:
This needs to be added to index.d.ts as a property on `ConsoleTransportOptions`.